lib/multiplexer/proxyline.go

```diff
@@ -0,0 +1,124 @@
+/**
+ *  Copyright 2013 Rackspace
```

Comment:
Do we have any other Rackspace code in Teleport? If not we need to send notifications of the additional license right?

Reply:
license is the same - apache2